packages/mobile/src/account/saga.ts

```diff
@@ -97,6 +99,10 @@ function* migrateAccountToProperBip39() {
   }
 }
 
+function* clearStoredAccount({ account }: ClearStoredAccountAction) {
+  yield call(removeAccountLocally, account)
```

Comment:
Do we also need to restart the app to send user back to onboarding or something like that?

Reply:
I am doing that on the `Settings` screen, navigating to `Screens.OnboardingEducationScreen`. The problem with that is that I'm not handling possible errors, this is related to what I wrote above.